src/SourceBuild/content/test/Microsoft.DotNet.UnifiedBuild.Tests/Exclusions.cs

```diff
@@ -8,7 +8,8 @@
 using System.IO;
 using System.IO.Enumeration;
 using System.Linq;
-using Microsoft.DotNet.SourceBuild.SmokeTests;
```

Comment:
Consider naming the folder `Microsoft.Dotnet.UnifiedBuild.BaselineTests` or create a separate directory within `Microsoft.Dotnet.UnifiedBuild.Tests` called `BaselineTests`. I think we have the SB folder named `Microsoft.DotNet.SourceBuild.SmokeTests` to match the namespace.

Reply:
Updated to `Microsoft.Dotnet.UnifiedBuild.BaselineComparison.Tests` everywhere.